phpcs.xml

```diff
@@ -23,10 +20,7 @@
     <!--  Include all Drupal related rules but exclude some so we can gradually fix these. -->
     <rule ref="Drupal">
         <exclude name="Drupal.Commenting.ClassComment.Short"/>
-        <exclude name="Drupal.Commenting.Deprecated.IncorrectTextLayout"/>
-        <exclude name="Drupal.Commenting.Deprecated.DeprecatedPeriodAfterSeeUrl"/>
-        <exclude name="Drupal.Commenting.Deprecated.DeprecatedWrongSeeUrlFormat"/>
-        <exclude name="Drupal.Arrays.Array.LongLineDeclaration"/>
     </rule>
+    <rule ref="DrupalPractice" />
```

Comment:
Can you add an explanation on what DrupalPractice is giving us in addition to Drupal?

I see the link, but I also know we previously didn't use it because it was quite opinionated and didn't always fit with our own opinions.  Documenting why we add it now can help us in discussions in the future where we may find that patterns we want to adopt move faster than Drupal does.

Reply:
DrupalPractice provides:

1. Array declaration line limits
2. Scanning to t() calls
3. \Drupal calls or things related to depending injections
4. It also suggests avoiding the use of global constants
5. It suggests addition of documentation for insecure routes in routing.yml
6. Unused variables

Also, the list is given here: https://www.drupal.org/project/coder/issues/2823933#comment-11757833

```
Detects the following bad practices:

    Unused and undefined PHP variables
    @author tags in doc blocks
    form_set_error() being used without t()
    'access administration pages' permission is used in hook_menu()
    Un-namespaced classes and interfaces are not prefixed with the module name
    $form_state['input'] is used instead of $form_state['values']
    $field['und'] is used instead of $field[LANGUAGE_NONE]
    Form API #options values are not translated with t()
    variable_get() calls where variables are not prefixed with the module name

```